ranger-http/src/main/java/io/appform/ranger/http/servicefinderhub/HttpServiceDataSource.java

```diff
@@ -28,13 +30,20 @@
 @Slf4j
 public class HttpServiceDataSource<T> extends HttpNodeDataStoreConnector<T> implements ServiceDataSource {
 
-    public HttpServiceDataSource(HttpClientConfig config, HttpCommunicator<T> httpClient) {
+    private final Set<String> excludedServices;
+
+    public HttpServiceDataSource(HttpClientConfig config, HttpCommunicator<T> httpClient,
+                                 final Set<String> excludedServices) {
         super(config, httpClient);
+        this.excludedServices = excludedServices;
```

Comment:
set this to Objects.requireNotNullElseGet(excludedServices, () -> Set.of())

Reply:
removed , not required